module-2/backend/week_three.md

```diff
@@ -10,14 +10,36 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+  rails g new app_name
 2. What do Models generally inherit from in rails?
+  ActiveRecord::Base
 3. What do Controllers generally inherit from in a rails project?
+  ActionController::Base
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  resources :horses only: [:view]
 5. What rake task is useful when looking at routes, and what information does it give you?
+  bundle exec rake routes
 6. What is an example of a route helper? When would you use them?
+  root_url or root_path 
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+  "_path" will only stay on the server, based upon the controller & router setup.  If we use the _url version, then we end up
+  giving/using the acutal IP address, server information, and a lot of other highly detailed information which is not generally
+  needed when building route helpers.
 8. What are strong params and why are the necessary?
+  they are used to establish paramerters which can & cannot be changed.  We establish which ones in our private methods, which
```

Comment:
strong params allow us to only accept certain parameters submitted from a form. we don't want to pass just any data to our database, we want to ensure we're only passing the parameters we want to allow our user's to update/create